packages/flutter/lib/src/widgets/autocomplete.dart

```diff
@@ -227,6 +261,28 @@ class RawAutocomplete<T extends Object> extends StatefulWidget {
   /// {@endtemplate}
   final AutocompleteOptionsViewBuilder<T> optionsViewBuilder;
 
+  /// {@template flutter.widgets.RawAutocomplete.loadingViewBuilder}
+  /// Builds a loading view while [optionsBuilder] is being resolved.
+  ///
+  /// The loading view is displayed floating below or above the field using a
+  /// [CompositedTransformFollower] inside of an [Overlay], not at the same
+  /// place in the widget tree as [RawAutocomplete]. To control whether it opens
+  /// upward or downward, use [optionsViewOpenDirection].
+  ///
+  /// {@endtemplate}
```

Comment:
Add "Defaults to null" and document what happens if you return null (e.g, does the widget show the previous options view when loading or nothing at all if this is null).